spec/models/metadata_source_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe MetadataSource, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Id normally ask for this to go, but given the follow on ticket of "add more specs" this seems fine